packages/jellyfish-api-core/src/category/account.ts

```diff
@@ -552,3 +561,13 @@ export interface BurnInfo {
    */
   paybacktokens: string[]
 }
+
+export interface GetFutureInfo {
+  owner: string
+  values: FutureData[]
+}
+
+export interface FutureData {
+  source: string
```

Comment:
```suggestion
  source: string // eg: '1.234@DUSD'
```
Am thinking should we stick to ain's naming too. (Although I think it is still confusing)
https://github.com/DeFiCh/ain/pull/1155/files#diff-84c7a7f36362b9724c31e5dec9879b2f81eae0d0addbc9c0933c3558c577de65R218-R221